templates/repo/commits_list.tmpl

```diff
@@ -61,11 +61,11 @@
 						{{end}}
 					</td>
 					{{if .Committer}}
-						<td class="text right aligned">{{DateUtils.TimeSince .Committer.When}}</td>
+						<td class="tw-text-right aligned">{{DateUtils.TimeSince .Committer.When}}</td>
 					{{else}}
-						<td class="text right aligned">{{DateUtils.TimeSince .Author.When}}</td>
+						<td class="tw-text-right aligned">{{DateUtils.TimeSince .Author.When}}</td>
```

Comment:
`aligned`?

Reply:
This seems to match on `.ui.table [class*="right aligned"]`.

Reply:
Actually it's just another instance of `text-align`:

```css
.ui.table[class*="right aligned"], .ui.table [class*="right aligned"] {
  text-align: right;
}

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/33905/commits/2e5246592cb2689856d6ce070cf8597c2b5358dc and removed another bunch of classes.